packages/agreement-email-sender/src/services/agreementEmailSenderService.ts

```diff
@@ -227,3 +139,266 @@ function getFormattedAgreementStampDate(
   }
   return dateAtRomeZone(new Date(Number(stampDate)));
 }
+
+async function sendAgreementActivationEmail(
+  emailManager: EmailManager,
+  readModelService: ReadModelService,
+  agreementV2Msg: AgreementV2,
+  templateService: HtmlTemplateService,
+  templateKind: AgreementEventMailTemplateType,
+  logger: Logger,
+  sender: { label: string; mail: string },
+  consumerName: string,
+  producerName: string,
+  recepientsEmails: string[],
```

Comment:
```suggestion
  recipientsEmails: string[],
```

Reply:
solved in [1b81944](https://github.com/pagopa/interop-be-monorepo/pull/1263/commits/1b81944476dd72c99885ed2b2f2f6d51d92e528e)